pytest_subtests.py

```diff
@@ -155,14 +156,24 @@ def _capturing_output(self):
                 fixture.close()
                 captured.out = out
                 captured.err = err
+    
+    @contextmanager
+    def _capturing_logs(self):
+        logging_plugin = self.request.config.pluginmanager.getplugin("logging-plugin")
```

Comment:
We should probably take in account that the logging plugin might be disabled via `-p no:logging`, so this call would return `None` and crash two lines below.

Could you account for that and include an additional test? Thanks!

Reply:
Good catch - I updated with one approach, let me know what you think.